SwiftSnake/SwiftSnake/Extensions/GameLogicControllers/PieceToDisplay.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PieceToDisplay.swift
+//  SwiftSnake
+//
+//  Created by Ellen Shapiro (Vokal) on 9/24/15.
+//  Copyright © 2015 Vokal. All rights reserved.
+//
+
+import Foundation
+
+enum PieceToDisplay {
+    case SnakeHead,
```

Comment:
I like that you don't have to add the word `case` to every line - I'm good with @chillpop's suggestion